### DISCUSSION TITLE
Welcome to gnucash-mobile Discussions!

### DISCUSSION THREAD

**Discussion by nicktylah** [nicktylah/gnucash-mobile#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seguinbg** [comment]
Guessi’m missing something
I can see account tree in app but
No way to enter a transaction

IPad latest iOS
Thx

**Reply by nicktylah** [reply]
Here's an example of creating a transaction. Latest iPad (simulator) on iOS 14.4

https://user-images.githubusercontent.com/6968226/110861902-a968ce00-8273-11eb-831e-c6aeeb68fd0c.mov

**Comment by seguinbg** [comment]
Lol

When I previously clicked on plus sign, I got some message
Repeated several times

Just tried again and it worked

Must have been that ufo that flew over earlier today
🤫

Thanks for responding



On Thu, Mar 11, 2021 at 5:13 PM Nick Tyler ***@***.***> wrote:

> Here's an example of creating a transaction. Latest iPad (simulator) on
> iOS 14.4
>
>
> https://user-images.githubusercontent.com/6968226/110861902-a968ce00-8273-11eb-831e-c6aeeb68fd0c.mov
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nicktylah/gnucash-mobile/discussions/1#discussioncomment-471761>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFKLQMN4BZB6E7A4JHDKIYLTDEW6VANCNFSM4USNPGPA>
> .
>


-- 
Bill

**Comment by fine-line** [comment]
Hello Nick. I for some reason can’t input numbers with decimal points in the amount field. It says please enter valid amount. Thank you for your work

**Reply by nicktylah** [reply]
Hmm, haven't been able to reproduce this on my end. Can you provide some more information, like your device, iOS version and the specific transaction amount you're attempting to enter?

**Comment by fine-line** [comment]
I am using iPhone SE iOS 14.4.2
here is a screenshot
<img width="320" alt="Снимок экрана 2021—05—19 в 23 05 21" src="https://user-images.githubusercontent.com/84065067/118877190-cca89d00-b8f6-11eb-8e23-7b7e4ab71143.png">
maybe it is because it is a comma and not a point? 
can it be related that I exported accounts in a different currency? In mobile app it is showing USD for some reason

**Reply by storyjesse** [reply]
Hi Nick, I'm not sure if I should post here or open an issue. I can't get the Mobile app to import accounts.
Steps to reproduce:

1. Installed GnuCash Mobile from GooglePlay store on Oppo A96 running Android Version 11 & ColorOS V11.1
2. On my laptop computer running GnuCash Version: 5.1 Build ID: Flathub 5.1 Finance::Quote: 1.5301
2.1 I export the account tree to CSV (File>Export>Export Account Tree to CSV...)
3. I send this file to my phone by attaching it to a telegram (saved message)
4. On My phone I open GnuCash Mobile and click "Import"
4.1 I use the filepicker to choose the Account-Tree.CSV file that I just transferred
...
5. I am presented with the GnuCash Mobile screen which still only shows the blue "Import" button. Nothing has changed, there is no account tree, no '+' button to click to create a new transaction. It's like the import failed silently.

Can you please help?

**Comment by MuchMX** [comment]
Hello.
I am a GnuCash desktop user on Mac iOS, and I would like to be able to enter data from the iPhone 13 mobile that I have.
Ican't install the app on iOs from App Store spain, I'm only offered the option to try the beta version on TestFlight, but I don't know if you have a beta version to try.
In that case I would need the invitation link to access it from TestFlight.
Or if the current version can be installed in some way from Spain.
Thank you very much.

MuchMX

**Comment by MuchMX** [comment]
I don't know Apple's policies for the applications it includes in one country or another, but the simplest thing (if possible) would be for the application to also be available in Spain through the AppStore.
Kind regards.
